src/SFML/Main/SFMLActivity.cpp

```diff
@@ -68,7 +68,7 @@ const char *getLibraryName(JNIEnv* lJNIEnv, jobject& objectActivityInfo)
     }
 
     // Convert the application name to a C++ string and return it
-    const jsize applicationNameLength = lJNIEnv->GetStringUTFLength(valueString);
+    const size_t applicationNameLength = static_cast<size_t>(lJNIEnv->GetStringUTFLength(valueString));
```

Comment:
```suggestion
    const std::size_t applicationNameLength = static_cast<std::size_t>(lJNIEnv->GetStringUTFLength(valueString));
```

We always qualify standard types with `std::`, please adjust other uses as well.

Reply:
Still, this is not targeted for C++11 unfortunately, yet.

Reply:
`std::size_t` is not a C++11 feature.

Reply:
Hm, you may be right.

My problem was that I wasn't sure if std::size_t was uniformly standardized in <cstddef> and whether this header existed on all versions of major compilers (as opposed to stddef.h). That's why I decided to go with the safer choice.

I'll change to std::size and see if all CI passes later.